drivers/base/kdgdb/amd64_sup.c

```diff
@@ -0,0 +1,263 @@
+/*
+ * COPYRIGHT:       GPL, see COPYING in the top level directory
+ * PROJECT:         ReactOS kernel
+ * FILE:            drivers/base/kddll/gdb_input.c
+ * PURPOSE:         Base functions for the kernel debugger.
```

Comment:
```suggestion
 * PROJECT:     ReactOS Kernel
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     Base functions for the kernel debugger
 * COPYRIGHT:   
```
Please, use the current header reactos.org/wiki/Coding_Style